pkg/cmd/release/create/create.go

```diff
@@ -342,16 +361,14 @@ func BuildPackageVersionBaseline(octopus *octopusApiClient.Client, deploymentPro
 				}
 
 				switch len(versions.Items) {
-				case 0:
-					// if channel rules have altered our query, tell the user about that
-					channelRulesHelp := ""
-					if query.PreReleaseTag != "" {
-						channelRulesHelp = fmt.Sprintf("%s, pre-release tag matching %s", channelRulesHelp, query.PreReleaseTag)
-					}
-					if query.VersionRange != "" {
-						channelRulesHelp = fmt.Sprintf("%s, version range matching %s", channelRulesHelp, query.VersionRange)
-					}
-					return nil, fmt.Errorf("no package version found for %s%s. please check that the package exists in your package feed", packageRef.PackageID, channelRulesHelp)
+				case 0: // no package found; cache the response
```

Comment:
Now that we consider "no available versions for this package" to be a legit thing, we need to cache that too